### PR TITLE
README - add link to instructions for getting pull secret

### PR DESCRIPTION
**Pull request by itewk**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -90,6 +90,7 @@ kind: QuayEcosystem
 metadata:
   name: example-quayecosystem
 spec:
+  imagePullSecretName: redhat-pull-secret
```
Comment (sabre1041) [inline README.md]: @itewk can you remove this reference to the `imagePullSecretName`?

**Review by sabre1041 — CHANGES_REQUESTED** [review]
Added one comment

File: README.md (hunk 1 above)
Comment (itewk) [reply README.md]: done.

**Comment by itewk** [comment]
@sabre1041 did the update.

**Review by sabre1041 — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**